Bottom: b8bd8de6044e86bb37b9827723a1b82a3fbbc83b
Top:    81aef8476e93d4a8584dfba4b8374a808e2196eb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 12:21:27 +0000

Refresh of nearly-working

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 8da586a..2137297 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1861,7 +1861,6 @@ cntr_Collect (jmp_buf * e, PLINE ** A, POLYAREA ** contours, PLINE ** holes,
     }
   else
     {
-      printf ("Got non ISECTED contour in cntr_Collect!\n");
       switch (action)
 	{
 	case PBO_ISECT:
